modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -1,40 +1,46 @@
-= Migrating User Key Encryption to Master Key Encryption
+= Migrating User Key-based Encryption to Master Key Encryption
 
-== Why Should I Move Away From User Key-based Encryption
+== Why Should I Move Away From User Key-based Encryption?
 
-NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. For new installations, Master key-based encryption is the recommended setup. The decryption workflow described here will only work when users have enabled password recovery and if an admin recovery password is available for the ownCloud administrator.
+NOTE: User key-based encryption is planned to be removed from ownCloud in the near future.
+While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages.
+It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups.
+Therefore Master-key-based encryption is now the recommended setup for all new installations.
 
-If you need to migrate from User Key-based to Master Key-based encryption, there are several steps that you need to follow to ensure a smooth and complete transition.
-These are:
+The decryption workflow described here works for both MySQL and SQLite based databases. It will only work when users have enabled the password recovery. Moreover, an admin recovery password needs to be activated and available for the ownCloud administrator.
```

Comment:
The second sentence should start with "However, ", given the preceding sentence.